Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py

```diff
@@ -735,6 +734,8 @@ def __init__(
         )
 
     def __enter__(self):
+        from mock import patch
+
```

Comment:
Something to keep in mind for the near future: We can almost certainly switch to using `unittest.mock` once we require Python 3.3+